azavea/azavea-branding-guide
# Fix missing download icons

## Overview

Restore download icons that were missing from the [Logo Downloads page](https://branding.azavea.com/logo-downloads/). See screenshot below.

![image](https://user-images.githubusercontent.com/128699/135155054-f7fa8afb-0bc6-43c1-934b-b34bf1c09bed.png)

### Demo

![image](https://user-images.githubusercontent.com/128699/135155436-2e668b0e-380e-4637-a271-d1f52c37660f.png)

### Notes

The proper classnames appear to have slipped through the cracks in #134. This PR fixes those, and strips out unused fontello code that was effectively deprecated as part of the PR.


## Testing Instructions

 * Run locally.
 * Ensure download icons appear on http://localhost:4000/logo-downloads/
 * Look through remainder of site to make sure icons render okay.
